src/mudlet.cpp

```diff
@@ -1966,8 +1966,15 @@ void mudlet::commitLayoutUpdates()
     while (itHostToolbarsList.hasNext()) {
         itHostToolbarsList.next();
         for (auto pToolBar : itHostToolbarsList.value()) {
-            if (Q_LIKELY(pToolBar) && pToolBar->property("layoutChanged").toBool()) {
-                pToolBar->setProperty("layoutChanged", QVariant(false));
+            // Under some circumstances there is NOT a
+            // pToolBar->property("layoutChanged") and examining that
+            // non-existant variant to see if it was true or false causes seg. faults!
```

Comment:
Shouldn't checking for a non-existing one just be a normal check, not a warning as well?

Reply:
I am not sure that it *should* be occurring - so until the cause for it has been established and fixed (🤞 ) I would want to have feedback on just how often/when it is happening...